rfd/0064-bot-for-cert-renewals.md

```diff
@@ -1,6 +1,6 @@
 ---
 authors: Zac Bergquist (zac.bergquist@goteleport.com), Tim Buckley (tim@goteleport.com)
-state: draft
+state: implemented
```

Comment:
We've received requests to start putting the version that the feature was released in.

Can you say `implemented (v10.0.0-v10.1.0)` or something here?

Reply:
For sure

Reply:
I've gone for v9.0.0 - v10.1.0 since that seems to be the range of versions we released the work described in this RFD over.